readme-en.md

```diff
@@ -14,69 +16,86 @@
 
 - [Projects](#projects)
 - [Software](#software)
+- [Templates](#templates)
 - [Tools](#tools)
-- [Courses](#courses)
+- [Guides](#guides)
+- [Public Courses](#public-courses)
 
 ## Projects
 
-- [directUC](https://github.com/wachunei/directUC) - Google Chrome and Opera extension for logging in to different services.
+Projects created to facilitate process university processes.
```

Comment:
A qué te refieres? Tal vez "Projects to facilitate university processes" nomás.

Reply:
Mala mía, fue in typo.